src/rgw/rgw_usage.cc

```diff
@@ -169,9 +169,9 @@ int RGWUsage::show(const DoutPrefixProvider *dpp, rgw::sal::Store* store,
 
   formatter->open_array_section("StorageStats");
   encode_json("size", stats.size, formatter);
-  encode_json("size_actual", stats.size_rounded, formatter);
-  encode_json("size_kb", rgw_rounded_kb(stats.size), formatter);
-  encode_json("size_kb_actual", rgw_rounded_kb(stats.size_rounded), formatter);
+  encode_json("size_actual", stats.size_utilized, formatter);
+  encode_json("size_kb", stats.size_rounded, formatter);
+  encode_json("size_kb_actual", rgw_rounded_kb(stats.size_utilized), formatter);
```

Comment:
why change in the common code.
They are displaying the correct thing r8?

Reply:
Addressed